src/components/TimeGraph.vue

```diff
@@ -53,6 +53,17 @@
           easyBiograph
         </div>
 
+        <div class="navbar-item">
+          <select
+              id="langselect"
+              name="lang"
+              v-model="lang"
+              @change="onLangChange($event)">
```

Comment:
v-model and @change both set the language -> evt. replace with a computed with setter